ziti/cmd/ascode/exporter/exporter.go

```diff
@@ -86,46 +91,43 @@ func NewExportCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 
 	v.AutomaticEnv()
 
-	edge.AddLoginFlags(cmd, &exporter.loginOpts)
+	edge.AddLoginFlags(cmd, &exporter.LoginOpts)
 	cmd.Flags().SetInterspersed(true)
-	cmd.Flags().BoolVar(&exporter.ofJson, "json", true, "Output in JSON")
-	cmd.Flags().BoolVar(&exporter.ofYaml, "yaml", false, "Output in YAML")
-	cmd.Flags().StringVar(&exporter.loginOpts.ControllerUrl, "controller-url", "", "The url of the controller")
+	cmd.Flags().BoolVar(&exporter.OfJson, "json", true, "Output in JSON")
+	cmd.Flags().BoolVar(&exporter.OfYaml, "yaml", false, "Output in YAML")
+	cmd.Flags().StringVar(&exporter.LoginOpts.ControllerUrl, "controller-url", "", "The url of the controller")
 	cmd.MarkFlagsMutuallyExclusive("json", "yaml")
 	ziticobra.SetHelpTemplate(cmd)
 
-	cmd.Flags().StringVarP(&exporter.filename, "output-file", "o", "", "Write output to local file")
-
-	exporter.loginOpts.Out = out
-	exporter.loginOpts.Err = errOut
+	cmd.Flags().StringVarP(&exporter.Filename, "output-file", "o", "", "Write output to local file")
 
 	return cmd
 }
 
 func (exporter *Exporter) Execute(input []string) error {
 
 	logLvl := logrus.InfoLevel
-	if exporter.loginOpts.Verbose {
+	if exporter.LoginOpts.Verbose {
 		logLvl = logrus.DebugLevel
 	}
 
 	pfxlog.GlobalInit(logLvl, pfxlog.DefaultOptions().Color())
 	internal.ConfigureLogFormat(logLvl)
 
 	var err error
-	exporter.client, err = exporter.loginOpts.NewMgmtClient()
+	exporter.client, err = exporter.LoginOpts.NewMgmtClient()
 	if err != nil {
 		return err
 	}
 
-	if exporter.filename != "" {
-		o, err := NewOutputToFile(exporter.loginOpts.Verbose, exporter.ofJson, exporter.ofYaml, exporter.filename, exporter.loginOpts.Err)
+	if exporter.Filename != "" {
```

Comment:
with the change to allow supplying an output stream, it feels like the output file/input file stuff could be refactored to be relevant to the command exclusively, not to the exporter/importers